variables.tf

```diff
@@ -33,18 +33,17 @@ variable "data_sources" {
   description = "The Azure resource IDs of existing resources that are required by the module."
   type = object({
     dns_zone_id = object({
-      cloud_aurora_ca = string
```

Comment:
Can you explain the removal of cloud_aurora_ca please? I think everything else looks good!

Reply:
I gave it  more explicit naming :)